code/game/machinery/doors/airlock.dm

```diff
@@ -100,9 +100,10 @@
 	var/security_level = 0 //How much are wires secured
 	var/aiControlDisabled = AI_WIRE_NORMAL //If 1, AI control is disabled until the AI hacks back in and disables the lock. If 2, the AI has bypassed the lock. If -1, the control is enabled but the AI had bypassed it earlier, so if it is disabled again the AI would have no trouble getting back in.
 	var/hackProof = FALSE // if true, this door can't be hacked by the AI
-	var/secondsMainPowerLost = 0 //The number of seconds until power is restored.
-	var/secondsBackupPowerLost = 0 //The number of seconds until power is restored.
-	var/spawnPowerRestoreRunning = FALSE
+	var/main_power_timer = 0 // Timer id, active when we are actively waiting for the main power to be restored
+	var/main_power_time // Paired with main_power_timer. Records its remaining time when something happens to interrupt power regen
+	var/backup_power_timer = 0 // Timer id, active when we are actively waiting for the backup power to be restored
+	var/backup_power_time // Paired with backup_power_timer. Records its remaining time when something happens to interrupt power regen
```

Comment:
```suggestion
	var/main_power_timer = 0 //! Timer id, active when we are actively waiting for the main power to be restored
	var/main_power_time //! Paired with main_power_timer. Records its remaining time when something happens to interrupt power regen
	var/backup_power_timer = 0 //! Timer id, active when we are actively waiting for the backup power to be restored
	var/backup_power_time //! Paired with backup_power_timer. Records its remaining time when something happens to interrupt power regen
```

you could also do the whole `///` style of dmdoccing but i just suggested it like this because i didn't want to spend the time doing that. just want to make sure it at least shows up statistically without really looking weird in comparison to the rest of the variables on this type

Reply:
I think I'll do ///, thanks